db/schema.rb

```diff
@@ -331,9 +331,9 @@
     t.uuid "customer_id"
     t.boolean "legacy", default: false, null: false
     t.float "vat_rate"
-    t.integer "status", default: 0, null: false
     t.bigint "credit_amount_cents", default: 0, null: false
     t.string "credit_amount_currency"
+    t.integer "status", default: 0, null: false
```

Comment:
You can rebuild your db to not have this change ;)

Reply:
ah yes wanted to remove the `schema.rb` from this PR scope